### PR TITLE
add feedback form

### PR DESCRIPTION
**Pull request by sebas-stripe**
I have a basic form that will POST to in this spreadsheet:

https://docs.google.com/spreadsheets/d/1VwUk6y1IggkLLbFZofJskr6He2zuY6kQWJgGmZ_PG8A/edit#gid=0

image

Any ideas on the following:

- Where should it go? Right now it is in the middle of the state page.
- Should it ask for email?
- Does it look ok? How should it be different?

TODO:

- [x] POST without a redirect
- [x] Have a success state

cc @koop-stripe

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,8 @@
 # Dependency directories
 node_modules/
 
+.node-version
```
Comment (koop) [inline .gitignore:11]: I don't think this should be in the gitignore? It'll be useful for us to run on a consistent node version (if this file isn't here already, I have one in my branch set to `12.16.1` and can merge it shortly)

**Comment by koop** [comment]
Overall, I think we'll want to make the "leave us feedback" button look more button-like, but let's start here and iterate 🙂

File: .gitignore (hunk 1 above)
Comment (sebas-stripe) [reply .gitignore:11]: will revove

**Comment by sebas-stripe** [comment]
Thanks!

I'll make it move button-like by adding borders to it. I still need to figure out where to put it!

Some options:

1. At the end of the content, inline with the content

image

2. Bottom right corner, fixed (and hidden in narrow viewports where it would be overlap with the content)

image

**Comment by sebas-stripe** [comment]
I rebased to master and made a bunch of changes. Here is how it works:

feedback

It also hides when the viewport is narrower and the button would overlap with the content. This means that there will be no feedback from mobile until I I find a mobile placement.

**Comment by sebas-stripe** [comment]
@koop I think this is ready to merge.

**Pull request merged**